docs/client_install/src/asciidoc/_chapters/dbviz.adoc

```diff
@@ -69,14 +66,16 @@ image:{images}/DbVisualizer_Driver_Manager.jpg[image]
 * Use the JDBC URL format:
 +
 ```
-jdbc:t4jdbc://<host-name or ip-address>:37800/:
+jdbc:t4jdbc://<host-name or ip-address>:23400/:
 ```
++
+*NOTE*: This example uses a modified port number (*37800*) rather than the default port number (*23400*).
```

Comment:
I don't get it. The example uses 23400, and then the *NOTE* says we aren't using 23400.